mediacapture-streams/obtaining-local-multimedia-content/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+<title>getUserMedia: test that getUserMedia is present (with or without vendor prefix)</title>
+<link rel="author" title="Dr Alex Gouaillard" href="mailto:agouaillard@gmail.com"/>
+<link rel="help" href="https://www.w3.org/TR/mediacapture-streams/#mediadevices-interface-extensions">
+<meta name='assert' content='Check that the mediaDevices.getUserMedia() method is present.'/>
+<meta name='flags' content='dom'/>
```

Comment:
I confirm it's gone in https://github.com/w3c/web-platform-tests/pull/2657/commits/a1aa788cf167b726deb48e47cbd6fb71568e7647